Services/Component/classes/class.ilPlugin.php

```diff
@@ -915,6 +921,13 @@ function update()
 		$ilCtrl->insertCtrlCalls("ilobjcomponentsettingsgui", ilPlugin::getConfigureClassName($this->getPluginName()),
 			$this->getPrefix());
 
+		// read the plugin.xml to get the event listening
+		include_once("Services/Component/classes/class.ilPluginReader.php");
```

Comment:
The include can be ommitted, just run a `composer dumpautoload` in libs/composer and the class will be part of the classmap and will be found in the autoloader